acceptance/help/output.txt

```diff
@@ -99,6 +99,7 @@ Vector Search
 Dashboards
   genie                                  Genie provides a no-code experience for business users, powered by AI/BI.
   lakeview                               These APIs provide specific management operations for Lakeview dashboards.
+  lakeview-embedded                      Token-based Lakeview APIs for embedding dashboards in external applications.
```

Comment:
Does this need to be regenerated now that we have a bumped SDK + new spec?

Reply:
Was the generated tree in the repo out of sync with the spec?

Reply:
Tbh not sure, the command was added in 0.241.0 but was not showing up in help output until this PR